src/pybind/mgr/telemetry/module.py

```diff
@@ -384,6 +384,7 @@ def compile_report(self, channels=[]):
                 report['pools'].append(
                     {
                         'pool': pool['pool'],
+                        'pool_name': pool['pool_name'],
```

Comment:
i am not sure if we should include the information which could be used to identify our users in the telemetry report.